src/index.html

```diff
@@ -7,6 +7,298 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="link" href="https://mate.academy/ru">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item" data-qa="nav-hover">
+            <a href="#laptops&computers" class="nav__link">Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <div class="product-card" data-qa="card">
+          <div class="product-card__image-wrapper">
+            <img
+              class="product-card__image"
+              src="/images/imac.jpeg"
+              alt="Product Image">
+          </div>
+          <h3 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="product-card__code">Product code: 195434</p>
+          <div class="product-card__rating-and-reviews">
+            <div class="product-card__stars product-card__stars--4">
+              <div class="product-card__star"></div>
+              <div class="product-card__star"></div>
+              <div class="product-card__star"></div>
+              <div class="product-card__star"></div>
+              <div class="product-card__star"></div>
+            </div>
+            <p class="product-card__reviews">Reviews: 5</p>
+          </div>
+          <div class="product-card__price-and-price-value">
+            <p class="product-card__price">Price:</p>
+            <p class="product-card__price-value">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="product-card__buy-button"
+            data-qa="card-hover">
+            Buy
+          </a>
+        </div>
+        <div class="product-card">
+          <div class="product-card__image-wrapper">
+            <img
+              class="product-card__image"
+              src="/images/imac.jpeg"
+              alt="Product Image">
```

Comment:
Need to check and fix it everywhere
```suggestion
            <img
              class="product-card__image"
              src="/images/imac.jpeg"
              alt="Product Image"
            >
```